src/lib/rose_tree/rose_tree.ml

```diff
@@ -15,15 +15,15 @@ let to_yojson conv t = display_to_yojson conv (to_display t)
 let of_yojson conv json =
   Result.map ~f:of_display (display_of_yojson conv json)
 
-let rec of_list_exn = function
+let rec of_list_exn ?(subtrees = []) = function
```

Comment:
Can you make the input of this be a `nonempty_list`? It result in less less `exn`s in our codebase.

Reply:
Sure, I can do that.